plugins/mention/trumbowyg.mention.js

```diff
@@ -0,0 +1,99 @@
+/* ===========================================================
+ * trumbowyg.mention.js v0.1
+ * Mention plugin for Trumbowyg
+ * http://alex-d.github.com/Trumbowyg
+ * ===========================================================
+ * Author : Viper
+ *          Github: https://github.com/Globulopolis
+ *          Website: http://киноархив.com
+ */
+
+(function ($) {
+    'use strict';
+
+    var defaultOptions = {
+        source: '',
+        formatDropdownItem: formatDropdownItem,
+        formatResult: formatResult
+    };
+
+    $.extend(true, $.trumbowyg, {
+        langs: {
+            en: {
+                mention: 'Mention'
+            },
+            ru: {
+                mention: 'Упомянуть'
+            }
+        },
+
+        plugins: {
+            mention: {
+                init: function (trumbowyg) {
+                    trumbowyg.o.plugins.mention = $.extend(true, {}, defaultOptions, trumbowyg.o.plugins.mention || {});
+
+                    var btnDef = {
+                        dropdown: buildDropdown(trumbowyg.o.plugins.mention.source, trumbowyg)
+                    };
+
+                    trumbowyg.addBtnDef('mention', btnDef);
+                }
+            }
+        }
+    });
+
+    /**
+     * Build dropdown list
+     *
+     * @param   {Array}   items      Items
+     * @param   {object}  trumbowyg  Editor
+     *
+     * @return  {Array}
+     */
+    function buildDropdown(items, trumbowyg) {
+        var dropdown = [];
+
+        // Check if source is an array
+        if (trumbowyg.o.plugins.mention.source.constructor === Array) {
```

Comment:
Should use `items`

Reply:
What means `items`?

Reply:
Your function argument :)

Reply:
`dropdown: buildDropdown(trumbowyg.o.plugins.mention.source, trumbowyg)`

`function buildDropdown(items, trumbowyg) {`

So here you should write:

`if (items.constructor === Array) {`